rest/taskana-rest-spring/src/main/java/pro/taskana/common/rest/TaskanaRestExceptionHandler.java

```diff
@@ -117,13 +117,22 @@ public class TaskanaRestExceptionHandler extends ResponseEntityExceptionHandler
           Map.entry(UnsupportedDatabaseException.ERROR_KEY, HttpStatus.INTERNAL_SERVER_ERROR),
           Map.entry(ERROR_KEY_UNKNOWN_ERROR, HttpStatus.INTERNAL_SERVER_ERROR));
 
-  @ExceptionHandler(MaxUploadSizeExceededException.class)
```

Comment:
Why did you change this? I couldn't find any info online about the change of @ExceptionHandler(MaxUploadSizeExceededException.class) connected to the spring update

Reply:
Hallo @ryzheboka ,

lässt man die Methode wie sie ist, kommt es zu dieser Exception beim Starten der Applikation:

```
aused by: java.lang.IllegalStateException: Ambiguous @ExceptionHandler method mapped for [class org.springframework.web.multipart.MaxUploadSizeExceededException]: {protected org.springframework.http.ResponseEntity pro.taskana.common.rest.TaskanaRestExceptionHandler.handleMaxUploadSizeExceededException(org.springframework.web.multipart.MaxUploadSizeExceededException,org.springframework.web.context.request.WebRequest), public final org.springframework.http.ResponseEntity org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler.handleException(java.lang.Exception,org.springframework.web.context.request.WebRequest) throws java.lang.Exception}
	at org.springframework.web.method.annotation.ExceptionHandlerMethodResolver.addExceptionMapping(ExceptionHandlerMethodResolver.java:114) ~[spring-web-6.1.5.jar:6.1.5]
	at org.springframework.web.method.annotation.ExceptionHandlerMethodResolver.<init>(ExceptionHandlerMethodResolver.java:78) ~[spring-web-6.1.5.jar:6.1.5]
```

Entfernt man die ursprüngliche Methode, funktionieren die Tests nicht mehr, da der Default von Spring greift. Deswegen musste ich die Methode hier überschreiben